std_msgs/Header header
uint32 seq
uint32 parent_seq
uint32 keyframe_seq
geometry_msgs/Pose global_T
bool transform_good
int32 tracker_status
PointTrackMessage[] points